### DISCUSSION TITLE
希望deepin&UOS默认安装wine环境

### DISCUSSION THREAD

**Discussion by widon1104** [linuxdeepin/developer-center#3556] [Features Request & Ideas | 特性请求 & 头脑风暴]
我最近安装了deepinV20.8，然后从星火商店安装了kindle（wine）这一个软件，但是kindle这个wine的软件无论如何都无法启动，最后发现需要sudo apt install wine来安装wine环境才能启动kindle，所以希望deepin&UOS能够默认安装wine这个程序，顺便把wine启动所需要的库安装好，免得用户安装wine软件无法启动的情况

**Comment by BLumia** [comment]
> 从星火商店安装了kindle（wine）这一个软件，但是kindle这个wine的软件无论如何都无法启动

星火商店依赖 wine 的应用应当恰当声明依赖，表示对 wine 的依赖才合理来着... cc @shenmo7192 瞅瞅？

**Comment by Decodetalkers** [comment]
> 我最近安装了deepinV20.8，然后从星火商店安装了kindle（wine）这一个软件，但是kindle这个wine的软件无论如何都无法启动，最后发现需要sudo apt install wine来安装wine环境才能启动kindle，所以希望deepin&UOS能够默认安装wine这个程序，顺便把wine启动所需要的库安装好，免得用户安装wine软件无法启动的情况

包要声明依赖内容，而不是要发行版预装，应该问打包的负责人，发行版应该尽量依赖少的东西，而且wine这东西需要32位环境

**Reply by BLumia** [reply]
+1。不过说起来，wine 8.0 起应该有望不再需要 multilib 了吧，虽然仍然是实验性 :joy:

**Reply by shenmo7192** [reply]
> +1。不过说起来，wine 8.0 起应该有望不再需要 multilib 了吧，虽然仍然是实验性 😂

目前我这边在等，星火wine的开发等几个版本才会准备跟进

话说，deepin-wine是怎么做到用系统文管打开文件选择的？

**Comment by shenmo7192** [comment]
请转星火交流平台（Rocket Chat）或星火论坛反馈<br/>这个包应该很久没人维护了，应该在星火wine之前就存在了<br/>是依赖不对是吧，我记下了，近期可能会发布更新修复<br/><br/><br/><html>
<head>
    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
</head>
<body>
<div id="imail_signature" class="ntes-signature">        <a href="https://dashi.163.com/projects/signature-manager/detail/index.html?ftlId=1&name=shenmo&uid=1422953823%40qq.com&iconUrl=https%3A%2F%2Fmail-online.nosdn.127.net%2Fsm80a7a5355e7d26f8f7b1045db291ee06.jpg&items=%5B%22%22%2C%22jifengshenmo%40outlook.com%22%2C%22%22%2C%22%22%2C%22%22%5D" style="display:block;background:#fff; max-width: 400px; _width: 400px;padding:15px 0 10px 0;text-decoration: none; outline:none;-webkit-tap-highlight-color:transparent;-webkit-text-size-adjust:none !important;text-size-adjust:none !important;">
            <table cellpadding="0" style='width: 100%; max-width: 100%; table-layout: fixed; border-collapse: collapse;color: #9b9ea1;font-size: 14px;line-height:1.3;-webkit-text-size-adjust:none !important;text-size-adjust:none !important;'>
                <tbody style="font-family: 'PingFang SC', 'Hiragino Sans GB','WenQuanYi Micro Hei', 'Microsoft Yahei', '微软雅黑', verdana !important; word-wrap:break-word; word-break:break-all;-webkit-text-size-adjust:none !important;text-size-adjust:none !important;">
                    <tr>
                            <td width="38" style="padding:0; box-sizing: border-box; width: 38px;">
                                <img width="38" height="38" style="vertical-align:middle; width: 38px; height: 38px; border-radius:50%;" src="https://mail-online.nosdn.127.net/sm80a7a5355e7d26f8f7b1045db291ee06.jpg" />
                            </td>
                            <td style='padding: 0 0 0 10px; color: #31353b;'>
                                <div style="font-size: 16px;font-weight:bold; width:100%; white-space: nowrap; overflow:hidden;text-overflow: ellipsis;">shenmo</div>
                            </td>
                    </tr>
                        <tr width="100%" style="font-size: 14px !important; width: 100%;">
                            <td colspan='2' style="padding:10px 0 0 0; font-size:14px !important; width: 100%;">
                                    <div style="width: 100%;font-size: 14px !important;word-wrap:break-word;word-break:break-all;"></div>
                                    <div style="width: 100%;font-size: 14px ***@***.***</div>
                                    <div style="width: 100%;font-size: 14px !important;word-wrap:break-word;word-break:break-all;"></div>
                                    <div style="width: 100%;font-size: 14px !important;word-wrap:break-word;word-break:break-all;"></div>
                                    <div style="width: 100%;font-size: 14px !important;word-wrap:break-word;word-break:break-all;"></div>
                            </td>
                        </tr>
                </tbody>
            </table>
        </a></div><br/><br/><br/><div class="ntes-mailmaster-quote" style="padding-top: 1px; padding-bottom: 1px" >
    <div style=" margin-top: 20px; margin-bottom: 12px; font-size: 14px; line-height: 1.25; color: #89898c; " >---- 回复的原邮件 ----</div>
    <div style=" margin-bottom: 12px; font-size: 13px; line-height: 1.25; color: #2c2c2e; padding: 2px; border-radius: 8px; background-color: #f0f0f0; " >
        <table width="100%" cellpadding="0" cellspacing="10" border="0">
            <tr>
                <td valign="top" style=" width: 4em; font-size: 13px; line-height: 1.25; color: #89898c; white-space: nowrap; " >发件人</td>
                <td valign="top" style=" font-size: 13px; line-height: 1.25; color: #2c2c2e; word-break: break-all; " ><a class="mail-from" style="color: #1c83eb; text-decoration: none" ***@***.***" >***@***.***&gt;</a></td>
            </tr>
            <tr>
                <td valign="top" style=" width: 4em; font-size: 13px; line-height: 1.25; color: #89898c; white-space: nowrap; " >日期</td>
                <td class="mail-date" valign="top" style=" font-size: 13px; line-height: 1.25; color: #2c2c2e; word-break: break-all; " >2023年01月28日 17:15</td>
            </tr>
            <tr style="">
                <td valign="top" style=" width: 4em; font-size: 13px; line-height: 1.25; color: #89898c; " >收件人</td>
                <td valign="top" style=" font-size: 13px; line-height: 1.25; color: #2c2c2e; word-break: break-all; " ><a class="mail-to" style="color: #1c83eb; text-decoration: none" ***@***.***" >***@***.***&gt;</a></td>
            </tr>
            <tr style="">
                <td valign="top" style=" width: 4em; font-size: 13px; line-height: 1.25; color: #89898c; " >抄送至</td>
                <td valign="top" style=" font-size: 13px; line-height: 1.25; color: #2c2c2e; word-break: break-all; " ><a class="mail-cc" style="color: #1c83eb; text-decoration: none" ***@***.***" >***@***.***&gt;</a>、<a class="mail-cc" style="color: #1c83eb; text-decoration: none" ***@***.***" >***@***.***&gt;</a></td>
            </tr>
            <tr>
                <td valign="top" style=" width: 4em; font-size: 13px; line-height: 1.25; color: #89898c; " >主题</td>
                <td class="mail-subject" valign="top" style=" font-size: 13px; line-height: 1.25; color: #2c2c2e; word-break: break-all; " >Re: [linuxdeepin/developer-center] 希望deepin&amp;UOS默认安装wine环境 (Discussion #3556)</td>
            </tr>
        </table>
    </div>
    <div><meta http-equiv="Content-Type" content="text/html; charset=utf-8"><p></p>
<blockquote>
<p dir="auto">我最近安装了deepinV20.8，然后从星火商店安装了kindle（wine）这一个软件，但是kindle这个wine的软件无论如何都无法启动，最后发现需要sudo apt install wine来安装wine环境才能启动kindle，所以希望deepin&amp;UOS能够默认安装wine这个程序，顺便把wine启动所需要的库安装好，免得用户安装wine软件无法启动的情况</p>
</blockquote>
<p dir="auto">包要声明依赖内容，而不是要发行版预装，应该问打包的负责人，发行版应该尽量依赖少的东西，而且wine这东西需要32位环境</p>

<p style="font-size:small;-webkit-text-size-adjust:none;color:#666;">—<br>Reply to this email directly, <a href="https://github.com/orgs/linuxdeepin/discussions/3556#discussioncomment-4803199">view it on GitHub</a>, or <a href="https://github.com/notifications/unsubscribe-auth/ALNH54AMYFY4K3HU3FNVUMTWUTPSTANCNFSM6AAAAAAUJQE7AM">unsubscribe</a>.<br>You are receiving this because you were mentioned.<img src="https://github.com/notifications/beacon/ALNH54BEMMYNKHLRJNEC4W3WUTPSTA5CNFSM6AAAAAAUJQE7AOWGG33NNVSW45C7OR4XAZNRIRUXGY3VONZWS33OINXW23LFNZ2KUY3PNVWWK3TUL5UWJTQAJFFH6.gif" height="1" width="1" alt=""><span style="color: transparent; font-size: 0; display: none; visibility: hidden; overflow: hidden; opacity: 0; width: 0; height: 0; max-width: 0; max-height: 0; mso-hide: all">Message ID: <span>&lt;linuxdeepin/developer-center/repo-discussions/3556/comments/4803199</span><span>@</span><span>github</span><span>.</span><span>com&gt;</span></span></p>
<script type="application/ld+json">[
{
***@***.***": "http://schema.org",
***@***.***": "EmailMessage",
"potentialAction": {
***@***.***": "ViewAction",
"target": "https://github.com/orgs/linuxdeepin/discussions/3556#discussioncomment-4803199",
"url": "https://github.com/orgs/linuxdeepin/discussions/3556#discussioncomment-4803199",
"name": "View Discussion"
},
"description": "View this Discussion on GitHub",
"publisher": {
***@***.***": "Organization",
"name": "GitHub",
"url": "https://github.com"
}
}
]</script></div>
</div>
</body>
</html>

**Comment by shenmo7192** [comment]
> 我最近安装了deepinV20.8，然后从星火商店安装了kindle（wine）这一个软件，但是kindle这个wine的软件无论如何都无法启动，最后发现需要sudo apt install wine来安装wine环境才能启动kindle，所以希望deepin&UOS能够默认安装wine这个程序，顺便把wine启动所需要的库安装好，免得用户安装wine软件无法启动的情况

请转星火交流平台或星火论坛反馈
该包很久没维护了，时间很早，近期我会调整下到新的打包标准，同时增加依赖
